contents/blog/the-posthog-array-1-33-0.md

```diff
@@ -0,0 +1,130 @@
+---
+date: 2022-02-28
+title: Array 1.33.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes", "Product updates"]
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: Introducing a full multivariate Experimentation suite, granular permissions on dashboards, Altinity Cloud support for ClickHouse, configuring your instance from the PostHog UI and 290+ improvements/fixes.
+---
+
+PostHog 1.33.0 introduces a full Experimentation suite, enabling you to run multivariate experiments to ensure you maximize value for your end users. Introducing also support for a managed ClickHouse infrastructure through Altinity Cloud, insight legends and insight permalinks. Plus, there are over 290+ improvements and bugfixes included!
+
+For Enterprise customers, we've also added the ability to set granular permissions on dashboards. 
+
+<blockquote class='warning-note'>
+<b>IMPORTANT!</b> From this version onwards, you'll need to make sure to run all async migrations on your instance after upgrading. The next version (1.34.0) will not run until async migrations are completed. Please check out the <a href="/docs/self-host/configure/async-migrations/overview" target="_blank">async migrations</a> docs for details.
```

Comment:
Fine as is though note: because the page existed on 1.32 too, it would be good to be super specific that the steps are: (1) upgrade to 1.33.0 (2) run the async migrations.

Reply:
updated